File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--importing-a-form-region-that-is-designed-in-outlook_4.cs
        void RefreshTaskListBox()
        {
            listBox1.Clear();
            Outlook.TaskItem tempTaskItem;
            String[] tempArray = olkTextBox3.Text.Split(new Char[] { '|' });

            foreach (string tempString in tempArray)
            {
                tempTaskItem = FindTaskBySubjectName(tempString);
                if (tempTaskItem != null)
                {
                    this.listBox1.AddItem(tempTaskItem.PercentComplete.ToString()
                        + "% Complete -- " + tempTaskItem.Subject, 
                            System.Type.Missing);
                }
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--importing-a-form-region-that-is-designed-in-outlook_4.vb
    Private Sub RefreshTaskListBox()
        listBox1.Clear()
        Dim tempTaskItem As Outlook.TaskItem
        Dim tempArray As [String]() = olkTextBox3.Text.Split(New [Char]() {"|"c})

        Dim tempString As String
        For Each tempString In tempArray
            tempTaskItem = FindTaskBySubjectName(tempString)
            If Not (tempTaskItem Is Nothing) Then
                Me.listBox1.AddItem(tempTaskItem.PercentComplete.ToString() & _
                    "% Complete -- " & tempTaskItem.Subject, System.Type.Missing)
            End If
        Next tempString
    End Sub